openmetadata-service/src/main/java/org/openmetadata/service/apps/NativeApplication.java

```diff
@@ -0,0 +1,16 @@
+package org.openmetadata.service.apps;
+
+import org.openmetadata.schema.entity.app.Application;
+import org.openmetadata.service.jdbi3.CollectionDAO;
+import org.quartz.Job;
+import org.quartz.JobExecutionContext;
+
+public interface NativeApplication extends Job {
+  void init(Application app, CollectionDAO dao);
```

Comment:
may be we should also pass SearchRepository?

Reply:
done